tests/core/pyspec/eth2spec/utils/ssz/ssz_impl.py

```diff
@@ -1,14 +1,18 @@
 from typing import TypeVar
 
 from remerkleable.basic import uint
-from remerkleable.core import View
+from remerkleable.core import Type, View
 from remerkleable.byte_arrays import Bytes32
 
 
 def serialize(obj: View) -> bytes:
     return obj.encode_bytes()
 
 
+def deserialize(typ: Type[View], data: bytes) -> View:
+    return typ.decode_bytes(data)
```

Comment:
At some point we added an alias `ssz_serialize` for `serialize` and used `ssz_serialize` in the specs to indicate it's SSZ helper. It was removed because it is not used anymore.

Maybe add an `ssz_deserialize` alias for `deserialize` for spec readability.

Reply:
Good idea. I made this commit https://github.com/ethereum/consensus-specs/pull/3976/commits/6416a562abb6085b87889e9c202cd3d2cb7d2b39. I did the following:

* Rename `{de,}serialize` to `ssz_{de,}serialize`.
* Make aliases which call `ssz_{de,}serialize`.